Cargo.lock

```diff
@@ -999,26 +1057,26 @@ dependencies = [
 
 [[package]]
 name = "ryu"
-version = "1.0.5"
+version = "1.0.9"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "71d301d4193d031abdd79ff7e3dd721168a9572ef3fe51a1517aba235bd8f86e"
+checksum = "73b4b750c782965c211b42f022f59af1fbceabdd026623714f104152f1ec149f"
 
 [[package]]
 name = "s3audit"
 version = "1.0.0"
```

Comment:
I'm guessing we'll bump to 1.1 or 2.0 after this?

Reply:
Probably 2.0, since it was a big internal change. It's nice to have a marker for that.